eth_account/_utils/transaction_utils.py

```diff
@@ -14,6 +21,9 @@
     is_rpc_structured_access_list,
 )
 
+RLPStructure = Tuple[Union[bytes, bytearray, ChecksumAddress], Tuple[Union[int, str]]]
```

Comment:
I think this can be:
```suggestion
RLPStructure = Tuple[Tuple[str, Tuple[Hash32, ...]], ...]
```